### DISCUSSION TITLE
How can I send an NFT with web3?

### DISCUSSION THREAD

**Discussion by 621625** [gochain/web3#267] [Q&A]
Hello,

I'm MB. A very nice and polite guy.

How can I send an NFT with web3?

Example:

I already have 10 NFT's of the same type with contract 0xabc in wallet 0xdef.

How can I send one of those NFT's to another wallet 0xghi?

**Comment by treeder** [comment]
Would be something along the lines of: 

```sh
web3 contract call --address 0xCONTRACT_ADDRESS --abi https://raw.githubusercontent.com/zeromintapp/contracts/main/erc721/ERC721WithURI.abi --function transferFrom 0xFROM 0xTO TOKEN_ID
```

**Reply by 621625** [reply]
I tried to follow your example like so:

web3 contract call --address 0x24Cd801e18Bf1EC584F3a4fE277B0c279E6d4CC0 --abi https://raw.githubusercontent.com/zeromintapp/contracts/main/erc721/ERC721WithURI.abi --function transferFrom 0x24Cd801e18Bf1EC584F3a4fE277B0c279E6d4CC0 0x58332f21b2c5b37ed590f4dff98f3d36609eba3e 4416362903532298566267812629111648309615883984914152037999396278235775893505

**Comment by 621625** [comment]
This is the result:

https://polygonscan.com/tx/0xe0d5dcf4d1050b9c1713cacd721d157e520d011af34b579ea3ae9883e71c840a

The transaction goes through but nothing happens and the NFT is not sent.
